### PR TITLE
Add last audited property to plan endpoint

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

File: src/main/java/org/opentripplanner/routing/edgetype/StreetEdge.java [hunk 1]
```diff
@@ -98,6 +99,12 @@ public NumericFieldSet getExtraNumericFields() {
     public void setExtraNumericFields(NumericFieldSet extraNumericFields) {
         this.extraNumericFields = extraNumericFields;
     }
+
+    public Date getLastAudited() {
```
Comment (flibbertigibbet) [inline src/main/java/org/opentripplanner/routing/edgetype/StreetEdge.java]: I understand this is just a starter modification, but we should keep NIH-specific things out of the `StreetEdge` class.  Maybe for now could put this function on the `PlanGenerator`, which also will need to have NIH-specific factored out once the structure is in place.

**Comment by flibbertigibbet** [comment]
This is going to add some NIH-specific things to some general classes, so we'll have to be careful to go find and factor those bits back out again later once a more generalized framework is in place.

+1

File: src/main/java/org/opentripplanner/routing/edgetype/StreetEdge.java (hunk 1 above)
Comment (CloudNiner) [inline src/main/java/org/opentripplanner/routing/edgetype/StreetEdge.java]: Ok, I'll move this line back to PlanGenerator. AFAIK there is already some NIH specific stuff where this method gets called. 

Will also add a TODO for clarity.

Comment (flibbertigibbet) [inline src/main/java/org/opentripplanner/routing/edgetype/StreetEdge.java]: The bench and toilet counts aren't really NIH-specific, as they come from OSM data, and do not modify existing behavior unless extra query params are used.  They should probably get pulled out or moved eventually, anyways.

**Pull request merged**